### ISSUE TITLE
fully_type_constants, bug

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2620]
TYPES should also trigger https://rules.abaplint.org/fully_type_constants/

```abap
TYPES barfoo(20).
DATA foobar(20).
```
